thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/datalayer/dto/AnomalyFeedbackDTO.java

```diff
@@ -62,6 +69,15 @@ public AnomalyFeedbackDTO setComment(String comment) {
     return this;
   }
 
+  public List<AnomalyFeedbackReason> getReasons() {
+    return ImmutableList.copyOf(reasons);
```

Comment:
protective copy on get but not on set. This is a bit confusing.
Overall you'll see the DTOs getters/setters are not protective and all fields are non final. This is mostly because of how jackson works.

I'm fine with protective but the setter should be protective then. 
I can't remember all gotchas with jackson, so if you're not sure I think not being protective in DTOs is ok for the moment.

Reply:
Got it. I will remove it.